### PR TITLE
Update interfaces/webrtc-stats.idl

### PR DESCRIPTION
**Pull request by autofoolip**
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/7998398/whatwg/idl/webrtc-stats.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/564827938

### CODE REVIEW

**Review by lukebjerring — COMMENTED** [review]
@foolip - you had been adding explicit legacy tests when interface members are being dropped. Is this a case where we want to add a negative test?

**Comment by foolip** [comment]
@lukebjerring this is a dictionary and more work to test for, so here I wouldn't add an historical test :)

File: interfaces/webrtc-stats.idl [hunk 1]
```diff
@@ -277,7 +277,6 @@ dictionary RTCIceCandidateStats : RTCStats {
              long priority;
              DOMString url;
              DOMString relayProtocol;
-             boolean deleted = false;
```
Comment (foolip) [inline interfaces/webrtc-stats.idl]: This is a real change, but dictionaries aren't tested at all so it doesn't affect results.

**Review by foolip — APPROVED** [review]
All whitespace except for the `deleted` dictionary member.

**Pull request merged**